README.md

```diff
@@ -1,54 +1,69 @@
+<picture align="center">
+  <img src="https://user-images.githubusercontent.com/45081667/227194559-57c73a9e-51df-4d42-b9f0-8b6dd011ae7b.png" alt="Header" >
+</picture>
+
 <p align="center">
   <a href="https://1password.com">
-    <img alt="1Password logo" src="https://user-images.githubusercontent.com/7430639/205342015-46801fd8-6701-482f-9da9-e21e7e39b3a1.svg" height="64">
-    <h3 align="center">1Password Shell Plugins</h3>
+    <h1 align="center">1Password Shell Plugins</h1>
   </a>
 </p>
 
 <p align="center">
-  Seamless authentication for every tool in your terminal. <sup><b><a href="#-beta-notice">[BETA]</a></b></sup>
+ <h4 align="center"> Authenticate any CLI with your fingerprint </h4>
 </p>
 
 <p align="center">
-  <a href="https://developer.1password.com/docs/cli/shell-plugins"><b>Available plugins</b></a> ·
-  <a href="https://developer.1password.com/docs/cli/"><b>1Password CLI docs</b></a> ·
-  <a href="https://developer.1password.com/docs/cli/shell-plugins/contribute"><b>Contribution docs</b></a>
+  <a href="https://developer.1password.com/docs/cli/shell-plugins/"><img alt="Get Started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37" /></a>
+  <a href="https://developer.1password.com/docs/cli/shell-plugins/contribute"><img alt="Build" src="https://user-images.githubusercontent.com/45081667/226941089-473be407-5417-48e3-8710-55b2d4ec761a.png" height="37" /></a>
```

Comment:
"Build"? Build what? The Go code?

Reply:
I'd say:
- `Contribute`
- `Contribute a Plugin`
- `Build Your Own Plugin`
- `Build a Plugin`

Reply:
I changed it to `Contribute`